ci/src/test/scala/es/weso/asio/ci/ContinuousIntegrationLauncher.scala

```diff
@@ -34,45 +34,85 @@ import scala.io.Source
 class ContinuousIntegrationLauncher extends AnyFlatSpec {
 
   // Lists of files that will be needed during the execution of the tests.
-  private final val ONTOLOGY_FILES      =   List[Source]()
-  private final val INSTANCE_FILES      =   List[Source]()
-  private final val SHAPE_EX_FILES      =   List[Source]()
-  private final val SHAPE_MAP_FILES     =   List[Source]()
-  private final val RES_SHAPE_MAP_FILES =   List[Source]()
+  private final var ONTOLOGY_FILES      =   List[Source]()
+  private final var INSTANCE_FILES      =   List[Source]()
+  private final var SHAPE_EX_FILES      =   List[Source]()
+  private final var SHAPE_MAP_FILES     =   List[Source]()
+  private final var RES_SHAPE_MAP_FILES =   List[Source]()
 
   /**
-   * TO DO...
+   * adding files to list from ../src
    */
   def populateOntologyFiles(): Unit = {
-    // TO DO...
+    val files = getListOfFiles("../src", ".ttl")
+    var list = List[Source]()
+    for (dir <- files){
+      println(dir)
+      list=list:+Source.fromFile(dir)
+    }
+    ONTOLOGY_FILES = list
```

Comment:
This should be not done, ONTOLOGY_FILES should be a value and therefore should not be reassigned.